res/qut logo.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please remove this file. It violates brand guidelines, and further, we need written permission to customize it.